src/main.rs

```diff
@@ -397,12 +402,19 @@ mod verify {
         err |= status("   ", &schain.cek, &schain.pek, quiet);
         err |= status("      ", &cchain.ask, &schain.cek, quiet);
         err |= status("         ", &cchain.ark, &cchain.ask, quiet);
-        println!("\n • = self signed, ⬑ = signs, •̷ = invalid self sign, ⬑̸ = invalid signs");
+
+        if !quiet {
+            println!("\n • = self signed, ⬑ = signs, •̷ = invalid self sign, ⬑̸ = invalid signs");
+        }
 
         if err as i32 == 0 {
             Ok(())
         } else {
-            exit(err as i32)
+            let certificate_error: std::io::Error = std::io::ErrorKind::InvalidData.into();
+            Err(error::Context::new(
+                "SEV/CA certificate verification failed.",
+                Box::new(certificate_error),
+            ))
```

Comment:
I generally try to not add strong punctuation when I'm using a string to describe an error because that leaves the door open for callers to format the output without having to fight against any punctuation you chose.

I.e., a caller could just do:

```rust
if let Err(e) = verify::cmd(blah) {
    eprintln!("Something horrible happened [{}], but I'm not giving up just yet.", e);
}
```

```suggestion
            Err(error::Context::new(
                "certificate chain verification failed",
                Box::<std::io::Error>::new(std::io::ErrorKind::InvalidData.into()),
            ))
```

and don't be afraid to bring in some `use` statements to avoid having to type out so many module and type namespaces.

Reply:
Yea, I see what you mean. I changed this in my latest commit.